metricbeat/docs/fields.asciidoc

```diff
@@ -5977,7 +6002,7 @@ type: keyword
 *`azure.billing.product`*::
 +
 --
-The product type
+TProduct name for the consumed service or purchase. Not available for Marketplace.
```

Comment:
Is `TProduct` some Azure-domain term or a type? I could not find anything.

Reply:
I'm sorry @rdner! It's a typo. My bad!